app/images/icons/collapse.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect width="16" height="16" fill="#E5E5E5"/>
+<g clip-path="url(#clip0_0_1)">
+<rect width="413" height="679" transform="translate(-57 -402)" fill="white"/>
+<g filter="url(#filter0_d_0_1)">
+<path d="M-31 -376C-31 -380.418 -27.4183 -384 -23 -384H321C325.418 -384 329 -380.418 329 -376V252C329 256.418 325.418 260 321 260H-23C-27.4183 260 -31 256.418 -31 252V-376Z" fill="white"/>
+</g>
+<circle cx="8" cy="8" r="7.5" fill="#F2F3F4" stroke="#D6D9DC"/>
+<line x1="4.1001" y1="7.83325" x2="11.7668" y2="7.83325" stroke="#D6D9DC" stroke-linecap="round"/>
+</g>
+<rect x="-2918.5" y="-1621.5" width="9747" height="4874" stroke="black"/>
+<defs>
+<filter id="filter0_d_0_1" x="-58" y="-409" width="414" height="698" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+<feFlood flood-opacity="0" result="BackgroundImageFix"/>
+<feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+<feOffset dy="2"/>
+<feGaussianBlur stdDeviation="13.5"/>
+<feColorMatrix type="matrix" values="0 0 0 0 0.12594 0 0 0 0 0.182502 0 0 0 0 0.305378 0 0 0 0.26 0"/>
+<feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_0_1"/>
+<feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow_0_1" result="shape"/>
+</filter>
+<clipPath id="clip0_0_1">
+<rect width="413" height="679" fill="white" transform="translate(-57 -402)"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
Minified version:

```suggestion
<svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill="#E5E5E5" d="M0 0h16v16H0z"/><g clip-path="url(#a)"><path fill="#fff" d="M-57-402h413v679H-57z"/><g filter="url(#b)"><path d="M-31-376a8 8 0 0 1 8-8h344a8 8 0 0 1 8 8v628a8 8 0 0 1-8 8H-23a8 8 0 0 1-8-8v-628Z" fill="#fff"/></g><circle cx="8" cy="8" r="7.5" fill="#F2F3F4" stroke="#D6D9DC"/><path stroke="#D6D9DC" stroke-linecap="round" d="M4.1 7.833h7.667"/></g><path stroke="#000" d="M-2918.5-1621.5h9747v4874h-9747z"/><defs><clipPath id="a"><path fill="#fff" transform="translate(-57 -402)" d="M0 0h413v679H0z"/></clipPath><filter id="b" x="-58" y="-409" width="414" height="698" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="2"/><feGaussianBlur stdDeviation="13.5"/><feColorMatrix values="0 0 0 0 0.12594 0 0 0 0 0.182502 0 0 0 0 0.305378 0 0 0 0.26 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_0_1"/><feBlend in="SourceGraphic" in2="effect1_dropShadow_0_1" result="shape"/></filter></defs></svg>
```

Reply:
Actually, this will get minified by our build scripts. So might be best to leave it as is for readability.